null_safety_examples/cookbook/testing/integration/scrolling/pubspec.yaml

```diff
@@ -0,0 +1,21 @@
+name: scrolling
+description: A new Flutter project.
+
+environment:
+  sdk: '>=2.12.0 <3.0.0'
```

Comment:
Fun question going forward will be when to uplift the minimum to 2.13.0 and the like. I'm guessing this sample can't take advantage of the type aliases so 2.12.0 is fine as is.